website/landingpage/dev/src/pages/index.html

```diff
@@ -113,4 +113,24 @@ <h2 class="title-2 has-big-margin">From startups to the largest companies</h2>
       {{> whoisusingluigi}}
     </div>
   </div>
+</section>
+
+<section class="s-gradient">
+  <div class="row">
+    <div class="column small-12 large-6 content">
+      <div class="sub-title has-number">
+        <span class="text">Our Youtube Channel</span>
+      </div>
+      <h3 class="title-3">Get the latest from Luigi</h3>
+      <p class="description">
+        Always stay up too date with our growing collection of news, innvovations and video tutorials.
+      </p>
+      <p>
+        <a class="btn-primary" href="https://www.youtube.com/channel/UC5WsYsHapDlg2K3iXS4n4AQ" target="_blank">Browse Channel</a>
+      </p>  
+    </div>
+    <div class="column small-12 large-6 image">
+      <img src="{{root}}assets/img/youtube.svg" alt="Our Youtube Channel" />
```

Comment:
Could you please doublecheck with Olga regarding `alt` text here, because I think the logic was to have something like "Luigi - [name of section] - [name of illustration]", but I'm not sure :)